draft-holmberg-mmusic-t140-usage-data-channel.xml

```diff
@@ -318,24 +318,51 @@ Answer:
 
     <section anchor="sec.gw" title="Gateway Considerations">
       <t>
-        Multiple transport mechanisms have been defined for T.140 <xref target="T140"/>, due to 
-        the long history and usage of the service in legacy packet-switched and 
-        circuit-switched networks. Some examples are listed below:
-        <list style="symbols">
-          <t>IP text telephony in text conversation mode <xref target="RFC4103"/></t>
-          <t>IP text telephony in text relay mode, also known as Text-over-IP (ToIP) <xref target="V151"/></t>
-          <t>IP text telephony in text pass-through mode, also known as Voiceband-over-IP (VBDoIP) <xref target="V152"/></t>
-          <t>PSTN text telephony</t>
-        </list>
+        A number of real-time text transports and protocols have been defined for
+        both packet switched and circuit switched networks. Many are based on the 
+        ITU-T T.140 protocol on application and presentation level <xref target="T140"/>. 
+        At the time of writing this document, some mechanisms are no longer used, as
+        the technologies they use have been obsoleted etc, while others are still in use.
       </t>
       <t>
-        In addition to simply moving text between two transport mechanisms, a gateway
-        might have to perform procedures related to events like inactivity of T.140 traffic,
-        RTP packets received out of order and loss of incoming RTP packets.
+        When performing interworking between T.140 data channels and another real-time text
+        transports and protocols with real-time text in another, a number of factors 
+        need to be considered.
+        At the time of writing this document, the most common IP-based real-time text transport
+        is the RTP based mechanism defined in <xref target="RFC4103"/>. While this document
+        does not define a complete interworking solution, this list below provides some guidance
+        and considerations to take into account when designing an gateway for interworking
+        between T140 data channels and RTP-based T.140 transport:
+        <list style="symbols">
+          <t>
+            For each T.140 data channel there is an RTP stream for real-time text <xref target="RFC4103"/> . 
+            Redundancy is by default declared and used on RTP stream. On the T.140 data channel
+            there is no redundancy, but the reliable property <xref target="I-D.ietf-mmusic-data-channel-sdpneg"/>
+            of T.140 the data channel is set.
+          </t>
+          <t>
+            During a normal text flow, T140blocks received from one network are forwarded towards the other network.
+            Keep-alive traffic is implicit on the T140 data channel. <xref target="RFC4103"/> does not define
+            keep-alive traffic for the RTP stream.
```

Comment:
Keep-alive traffic is usually needed on the RTP side, even if RFC 4103 does not mention it. 
Proposal: replace the sentence about does not define.   "Keep-alive traffic MAY need to be extracted from incoming RTP traffic and generated on outgoing RTP traffic according to procedures used on the RTP side."

Reply:
Will fix.